cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala

```diff
@@ -46,9 +46,9 @@ class WhereTest extends DocumentingTest {
         | ** <<filter-on-dynamic-property, Filter on dynamically-computed property>>
         | ** <<property-existence-checking, Property existence checking>>
         |* <<query-where-string, String matching>>
-        | ** <<match-string-start, Match the beginning of a string>>
-        | ** <<match-string-end, Match the ending of a string>>
-        | ** <<match-string-contains, Match anywhere within a string>>
+        | ** <<match-string-start, Prefix search using `STARTS WITH`>>
```

Comment:
I thought the older text read better, and also included the type requirement (String). The new one is more subtle in the type requirement, since `prefix` implies `string`, but not as explicit about it. If people search the docs are they more like to search for `prefix` or `beginning of`?

Reply:
I've compromised: you're absolutely right in that the string-ness was lost. Have reinstated that part. Have left prefix, suffix etc (as they're more in line with existing ways of referring to this and more technical/terse), as even if the user searches for "beginning of", they will find it as it is mentioned thus in the description.

Reply:
👍